### ISSUE TITLE
Question about Retraining checkpoints

### ISSUE THREAD

**Issue by chaos2334** [cjw2021/QAHOI#18]
Hello, my average accuracy is only 19 when I train to 150 rounds. How can I continue to train based on the current results？

**Comment by cjw2021** [comment]
Thanks for your interests.
If you follow the training setting we provide, the result will be close to the one in the paper.
If you want to resume from a current checkpoint, you can use the `resume` argument.
```bash
python -m torch.distributed.launch \
        --nproc_per_node=8 \
        --use_env main.py \
        --backbone [backbone_name] \
        --resume [last checkpoint] \
        --output_dir [output directory] \
        --epochs [new epoch] \
        --lr_drop [drop at] \
        --num_feature_levels 3 \
        --num_queries 300 \
        --use_nms
```
